src/main/java/edu/hawaii/its/groupings/access/AuthorizationService.java

```diff
@@ -17,13 +16,16 @@
 @Service
 public class AuthorizationService {
 
-    @Autowired
-    GroupingsRestController groupingsRestController;
+    private final GroupingsRestController groupingsRestController;
 
-    @Autowired
-    UhUuidCheckerService uhUuidCheckerService;
+    private final UhUuidCheckerService uhUuidCheckerService;
     private static final Log logger = LogFactory.getLog(AuthorizationService.class);
```

Comment:
Can you move the logger to the top like we do in the other classes. Also, I'm not sure if we want the blank lines between the class-level variables, but whatever we decide is okay with me.